json/es.json

```diff
@@ -1,71 +1,71 @@
 {
-    "30 Days": "30 Days",
-    "60 Days": "60 Days",
-    "90 Days": "90 Days",
+    "30 Days": "30 Dias",
+    "60 Days": "60 Dias",
+    "90 Days": "90 Dias",
     ":amount Total": ":amount Total",
-    ":resource Details": ":resource Details",
-    ":resource Details: :title": ":resource Details: :title",
+    ":resource Details": "Detalles del :resource",
+    ":resource Details: :title": "Detalles del :resource : :title",
     "A fresh verification link has been sent to your email address.": "Se ha enviado un nuevo enlace de verificación a su correo electrónico.",
     "A new verification link has been sent to the email address you provided during registration.": "Se ha enviado un nuevo enlace de verificación a la dirección de correo electrónico que proporcionó durante el registro.",
     "Accept Invitation": "Aceptar invitación",
-    "Action": "Action",
-    "Action Happened At": "Happened At",
-    "Action Initiated By": "Initiated By",
-    "Action Name": "Name",
-    "Action Status": "Status",
-    "Action Target": "Target",
-    "Actions": "Actions",
-    "Add": "Agregar",
-    "Add a new team member to your team, allowing them to collaborate with you.": "Agregue un nuevo miembro a su equipo, lo que le permitirá colaborar con usted.",
+    "Action": "Acción",
+    "Action Happened At": "La acción sucedió a las",
+    "Action Initiated By": "La acción inició a las",
+    "Action Name": "Nombre de la acción",
+    "Action Status": "Estado de la acción",
+    "Action Target": "Objetivo de la acción",
+    "Actions": "acciones",
+    "Add": "Añadir",
+    "Add a new team member to your team, allowing them to collaborate with you.": "Agregue un nuevo miembro a su equipo, permitiendole colaborar con usted.",
     "Add additional security to your account using two factor authentication.": "Agregue seguridad adicional a su cuenta mediante la autenticación de dos factores.",
-    "Add row": "Add row",
-    "Add Team Member": "Agregar nuevo miembro al equipo",
-    "Added.": "Agregado.",
+    "Add row": "Añadir fila",
+    "Add Team Member": "Añadir un nuevo miembro al equipo",
+    "Added.": "Añadido.",
     "Administrator": "Administrador",
     "Administrator users can perform any action.": "Los administradores pueden realizar cualquier acción.",
     "Afghanistan": "Afghanistan",
-    "Aland Islands": "Åland Islands",
+    "Aland Islands": "Islas Aland",
```

Comment:
Country names do not have to be translated into your language. This can only be done if you need names in your language.

Reply:
I prefer to translate countries. I prefer "Espagne" than "Spain", for example

Reply:
I understood. Look, when adding new keys, they are automatically added to all localizations, but since they are specified in the file for ignore, their translation is not required.

For example, in Russian localization, it is not at all necessary to translate country names into Russian. We often use English versions.

Reply:
If someone doesn't want to translate the name of a country they just can put the name outside the `__()` I mean if you write:

```blade
{{__('Country')}}: {{__('Mexico')}}
```
Into Spanish it should render (Notice the **é** is not **e**):
```
País: México
```

If you don't want to translate it, you should do:

```blade
{{__('Country')}}: Mexico
```

So it would render:
```
País: Mexico
```

There is **literally no point** making a translation file that does not translate

Reply:
I disagree. It is a bad idea to prescribe names in the code, because someone needs a translation, and someone does not.
For example, in Russian-language localization, the list of countries is rarely translated into Russian. Therefore, "Mexico" is displayed as "Mexico".

Reply:
If the Russian language has it's grammar exceptions that makes it untranslatable, that should be only on the Russian json file. not because English→Russian is untranslatable it means English→Spanish is not.

Reply:
It seems to me that we did not understand each other.
I am saying that translation is not required if you do not need it.

Reply:
actually if Russian can't translate, the ´ru.json´ shouldn't have the country key on the first place.

Reply:
In Spanish you need to translate the country names. As someone that has lived on Mexico and Spain for years I ensure you must Spanish speakers only know those country names in Spanish.

Reply:
Yes and no.

One of the old problems with this project is that the translation files contain outdated keys, which causes the problem of their validation. We have recently introduced the functionality of automatic key checking in all localizations. This code removes unused keys and adds missing ones.

Reply:
If someone is writing a code and write 
```blade
{{__('Country')}}: {{__('Spain')}}
```
Give me a reason for the **Spanish** render (A.K.A. not Russian) it should render:
```
País: Spain
```
instead of
```
País: España
```
In Spanish translating the country name makes a lot of sense.

Reply:
Are we discussing Taylor's solution to add code.

I repeat once again: if you want to translate, translate. Nobody forbids.

I see no reason to discuss this topic.